compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -5637,6 +5641,12 @@ private void transformDocumentationBacktickContent(Node backtickContent,
             default:
                 throw new IllegalArgumentException("Invalid backtick content transformation");
         }
+        if (bLangRefDoc.identifier != null) {
```

Comment:
Hope this is clean

Reply:
Looks good. 
btw I think we have to do this for `bLangRefDoc.qualifier` as well.
e.g. `'int:'β`